libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/AuthenticatorService.java

```diff
@@ -122,24 +118,16 @@ public Bundle addAccount(AccountAuthenticatorResponse response,
                         String authTokenType,
                         String[] requiredFeatures,
                         Bundle options) {
-        	if (isAddFromSettings(options)) {
-        		options.putAll(SalesforceSDKManager.getInstance().getLoginOptions().asBundle());
-        	}
         	return makeAuthIntentBundle(response, options);
         }
 
-        private boolean isAddFromSettings(Bundle options) {
-			return options.containsKey(ANDROID_PACKAGE_NAME)
-                    && SETTINGS_PACKAGE_NAME.equals(options.getString(ANDROID_PACKAGE_NAME));
-		}
-
         @Override
         public Bundle getAuthToken(AccountAuthenticatorResponse response, Account account,
                             String authTokenType, Bundle options) throws NetworkErrorException {
-
             UserAccount originalUserAccount = UserAccountManager.getInstance().buildUserAccount(account);
-            final Map<String,String> addlParamsMap = SalesforceSDKManager.getInstance().getLoginOptions().getAdditionalParameters();
+
             try {
+                final Map<String,String> addlParamsMap = originalUserAccount.getAdditionalOauthValues();
```

Comment:
This also comes up a couple times but I think it is a bug nobody ever caught?  The additional params that were stored on `LoginOptions` were _login_ params like `login_hint=username` added to the URL for the browser to interpret.  Here we appear to be adding those params to the token refresh call instead of the additional auth params stored on the user account that we get back from the server upon login.